### ISSUE TITLE
Footnotes impede reading when JS is disabled

### ISSUE THREAD

**Issue by ap** [Ovid/ovid.github.com#19]
When JS is disabled all footnotes are shown and the page overlay is active (same as when you click a footnote) with no way of disabling that.

Obviously, ideally footnotes would fall back to some way of reading them even without JS – but I’m not even asking for as much: it would just be nice if the page were readable without Reader mode or futzing around with the DOM inspector.

**Comment by Ovid** [comment]
Sorry about that, Aristotle. I'll give some thought on how best to deal with it.

**Comment by Ovid** [comment]
Hi @ap. Sorry it took me so long to get around to this. I believe this is fixed now. Feel free to reopen the ticket if it's not!

**Comment by ap** [comment]
Can confirm. Thanks!

I don’t know if the no-JS rendering of footnotes is as intended – it took me a while to even spot them at the far bottom of the page, even below all the usual footers, but they are in fact there. I mention this just because it looks sorta broken and maybe you care about that; as far as I care it’s entirely serviceable already.

**Comment by Ovid** [comment]
Yeah, it's not great. If you click on the footnote, `[1]` and friends, it takes you directly to the footnote via an anchor. I know it's ugly. Maybe I'll try to fix it. You might be the only person reading it without Javascript :)

**Comment by ap** [comment]
Yeah, I’m aware I’m an outlier. It’s functional enough now to invest no further effort if you don’t care to.
